src/adapter.rs

```diff
@@ -111,10 +101,14 @@ impl Adapter {
         if result.is_null() {
             Err("Failed to create adapter".into())
         } else {
+            let luid = crate::ffi::alias_to_luid(&name_utf16)?;
+            let index = crate::ffi::luid_to_index(&luid)?;
             Ok(Arc::new(Adapter {
                 adapter: UnsafeHandle(result),
                 wintun: wintun.clone(),
                 guid,
+                index,
+                luid,
             }))
         }
     }
```

Comment:
We want to delete [this comment](https://github.com/nulldotblack/wintun/blob/5c801d06983bceef814a060e530d3064aebcdd1a/src/adapter.rs#L124-L129) since now the index is always known now!

Reply:
Here too: https://github.com/nulldotblack/wintun/blob/5c801d06983bceef814a060e530d3064aebcdd1a/src/adapter.rs#L88-L90